expire-old-content.js

```
@@ -0,0 +1,206 @@
+    try {
+        content = fs.readFileSync(componentPath, 'utf8');
+    } catch (error) {
+        console.error(`Error reading component file ${componentPath}:`, error);
+        return;
+    }
+    let removedImages = new Set();
+
+    // Collect future images that should be temporarily removed
+    content.replace(/<img [^>]*src=\{([^}]+)\}[^>]*valid-from="([^"]+)"[^>]*>/g, (match, imgVar, dateStr) => {
+        const startDate = new Date(dateStr);
+        if (!isNaN(startDate.getTime()) && startDate > today) {
```

Biome

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)